packages/ckeditor5-source-editing/src/sourceediting.ts

```diff
@@ -369,6 +370,15 @@ export default class SourceEditing extends Plugin {
 		// Checks, if the editor's editable belongs to the editor's DOM tree.
 		return editable && !editable.hasExternalElement;
 	}
+
+	/**
+	 * If any {@link module:ui/dialog/dialogview~DialogView editor dialog} is currently visible, hide it.
+	 */
+	private _hideVisibleDialog(): void {
+		if ( this.editor.plugins.has( 'Dialog' ) && this.editor.plugins.get( 'Dialog' ).isOpen ) {
```

Comment:
See my other comment about type inference in `plugins.get`. TL;DR have typed variable.